src/components/Accordion.js

```diff
@@ -92,6 +99,16 @@ export const Accordion = props => {
       width="100%"
       {...rest}
     >
+      {position === "top" && (
+        <Animated.View
```

Comment:
@injas-bigbinary We can move out this code. Currently the same code written here and below. Rest looks good to me.